internal/settings/settings.go

```diff
@@ -45,6 +45,12 @@ const (
 )
 
 var (
+	// ConfigFilenames are all possible config filenames.
+	ConfigFilenames = []string{
+		DefaultConfigFilename,
```

Comment:
It looks like we're using this for `prototool config init`. I imagine that we'll need to support a `prototool config init --json` now that we support `prototool.json`. 

Not actionable in this change, but worth noting for now.

Reply:
I almost wanted to not document the `prototool.json` stuff as much for now, but thought that was probably not a good idea. I think that we should say we prefer `prototool.yaml` if you're generating a file, and `prototool.json` is sort of the "secondary" format, so if you use `prototool config init`, it only does YAML - at least I think this is fair for now.

Tldr: We're in agreement, maybe add later, heh.